### PR TITLE
https://issues.liferay.com/browse/LPS-77751

### PR DESCRIPTION
**Pull request by willnewbury**
Resend of confirmation boxes, with pre-translation

### CODE REVIEW

**Comment by willnewbury** [comment]
ci:retest

**Comment by drewbrokke** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/portlet/action/AutoAnonymizeUserAssociatedDataEntityMVCActionCommand.java [hunk 1]
```diff
@@ -51,6 +51,10 @@ protected void doProcessAction(
 
 		uadEntityAnonymizer.autoAnonymize(
 			getUADEntity(actionRequest, uadRegistryKey));
+
+		String redirect = ParamUtil.getString(actionRequest, "redirect");
+
+		sendRedirect(actionRequest, actionResponse, redirect);
```
Comment (drewbrokke) [inline modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/portlet/action/AutoAnonymizeUserAssociatedDataEntityMVCActionCommand.java]: @willnewbury how are these `*MVCActionCommand` changes related to the kebab menu?

Comment (drewbrokke) [reply modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/portlet/action/AutoAnonymizeUserAssociatedDataEntityMVCActionCommand.java]: Nevermind - I see you added the redirect as a parameter below

**Comment by drewbrokke** [comment]
PR sent to https://github.com/jonmak08/liferay-portal/pull/1858 <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**